OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/AuthenticatedWebView.java

```diff
@@ -17,14 +17,15 @@
 import android.webkit.WebResourceResponse;
 import android.webkit.WebView;
 import android.widget.FrameLayout;
-import android.widget.ProgressBar;
 
 import androidx.annotation.DrawableRes;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.annotation.StringRes;
 import androidx.fragment.app.FragmentActivity;
 
+import com.google.android.material.progressindicator.CircularProgressIndicator;
+
```

Comment:
I think we don't need these files changes.

Reply:
update the references for code improvements.